src/main/java/seedu/address/commons/core/AddressbookMessages.java

```diff
@@ -3,12 +3,16 @@
 /**
  * Container for user visible messages.
  */
-public class AnakinMessages {
+public class AddressbookMessages {
```

Comment:
We should merge `AnakinMessages` and `Messages` and just use `Messages` from now on